src/main/java/com/synopsys/integration/alert/workflow/startup/component/AlertStartupInitializer.java

```diff
@@ -167,76 +167,78 @@ private void initializeConfiguration(final Collection<String> descriptorNames, f
         }
     }
 
-    private Set<ConfigurationFieldModel> createFieldModelsFromDefinedFields(final String descriptorName, final List<DefinedFieldModel> fieldsForDescriptor, Map<String, ConfigurationFieldModel> existingConfiguredFields) {
-        final Set<ConfigurationFieldModel> configurationModels = new HashSet<>();
+    private Set<ConfigurationFieldModel> createFieldModelsFromDefinedFields(String descriptorName, List<DefinedFieldModel> fieldsForDescriptor, Map<String, ConfigurationFieldModel> existingConfiguredFields) {
+        Set<ConfigurationFieldModel> configurationModels = new HashSet<>();
         logger.info("  ### Environment Variables ### ");
-        for (final DefinedFieldModel fieldModel : fieldsForDescriptor) {
-            final String key = fieldModel.getKey();
-            final String convertedKey = convertKeyToProperty(descriptorName, key);
-            final boolean hasEnvironmentValue = hasEnvironmentValue(convertedKey);
+        for (DefinedFieldModel fieldModel : fieldsForDescriptor) {
+            String key = fieldModel.getKey();
+            String convertedKey = convertKeyToProperty(descriptorName, key);
+            boolean hasEnvironmentValue = hasEnvironmentValue(convertedKey);
             logger.info("    {}", convertedKey);
             logger.debug("         Environment Variable Found - {}", hasEnvironmentValue);
             String defaultValue = null;
             if (existingConfiguredFields.containsKey(key)) {
-                defaultValue = existingConfiguredFields.get(key).getFieldValue().get();
+                Optional<String> fieldValue = existingConfiguredFields.get(key).getFieldValue();
+                if (fieldValue.isPresent()) {
+                    defaultValue = fieldValue.get();
+                }
```

Comment:
This was the actual change. The other changes were just removing unnecessary final's

Reply:
I'm finding a lot of our PR's are bloated because of this. We should get in the habit of pointing people where to focus until the final keywords are mostly removed.